app/config/variables.php

```diff
@@ -311,7 +311,7 @@
                 'name' => '_APP_STORAGE_ANTIVIRUS',
                 'description' => 'This variable allows you to disable the internal anti-virus scans. This value is set to \'enabled\' by default, to cancel the scans set the value to \'disabled\'. When disabled, it\'s recommended to turn off the ClamAV container for better resource usage.',
```

Comment:
We need to update the default value in the description

Reply:
Updated!